### PR TITLE
revert-removing-pod-priority-calico

### PR DESCRIPTION
**Pull request by calvix**

### CODE REVIEW

File: v_3_6_2/master_template.go [hunk 1]
```diff
@@ -153,6 +153,7 @@ write_files:
             - effect: NoExecute
               operator: Exists
           serviceAccountName: calico-node
+          priorityClassName: system-cluster-critical
```
Comment (r7vme) [inline v_3_6_2/master_template.go]: Please add this to calico manifest changes, otherwise it will be probably gone on nxt update.

Comment (calvix) [reply v_3_6_2/master_template.go]: Where is calico manifest changes?

**Comment by calvix** [comment]
> Can you please provide more info where it comes?

all our critical components like coredns, kubeproxy  and others  have this to ensure that they have priority in scheduling and also that they are not evicted when there is pressure on node

File: v_3_6_2/master_template.go (hunk 1 above)
Comment (kopiczko) [reply v_3_6_2/master_template.go]: Should we also add this in azure-operator manifests?

Comment (teemow) [reply v_3_6_2/master_template.go]: Yes definitely. We should make sure to use the same source for both imo. Do we have an issue for that already?

Comment (calvix) [reply v_3_6_2/master_template.go]: probably yes, I am kinda shady on where is that

Comment (calvix) [reply v_3_6_2/master_template.go]: using same source fro both can be tricky as azure uses completely different calico ( AFAIK)

Comment (teemow) [reply v_3_6_2/master_template.go]: then we maybe should put a big fat warning in here. and add a link to the one in the azure operator.

Comment (calvix) [reply v_3_6_2/master_template.go]: Yeah make sense

Comment (r7vme) [reply v_3_6_2/master_template.go]: Changes are tracked in comments here https://github.com/giantswarm/k8scloudconfig/blob/master/v_3_7_1/master_template.go#L25

We have calico in many places, so we should be careful, i found most reliable way is to track deviations from upstream manifests in comments. Better ideas are welcome

Comment (r7vme) [reply v_3_6_2/master_template.go]: https://github.com/giantswarm/k8scloudconfig/pull/457

**Pull request merged**